src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -64,8 +66,8 @@
 
     // These are elements common to most pages in our app
 
-    @FindBy(id = "statusMessagesToUser")
-    private WebElement statusMessage;
+    @FindBy(className = "statusMessage")
```

Comment:
`id` is more appropriate here. Although it does not make sense that `.statusMessage` is outside of `statusMessagesToUser` we can't control that.